templates/navigationTitle.jsx

```diff
@@ -1,19 +1,33 @@
 import React from 'react';
+import Adapt from 'core/js/adapt';
+import device from 'core/js/device';
 
 export default function NavigationTitle(props) {
   const {
-    title,
+    _hideForMobile,
     _useCourseTitle
   } = props;
 
-  return (
+  const hideForMobile = ( _hideForMobile && !device.isScreenSizeMin('medium') )
+    ? false
+    : true;
+
+  const title = ( _useCourseTitle )
+    ? Adapt.course.get('title')
+    : Adapt.course.get('_navigationTitle').title;
```

Comment:
I think this logic should probably go in the view/model rather than in the template?

Reply:
You can put it in here if you want?

https://github.com/cgkineo/adapt-navigationTitle/blob/17f41f7cc0296278225641a7b3dce93e02444d23/js/adapt-navigationTitle.js#L12-L14
```js
 static get courseConfig() { 
   const config = Adapt.course.get('_navigationTitle'); 
   const hideForMobile = ( _hideForMobile && !device.isScreenSizeMin('medium') )
    ? false
    : true;
   const title = ( _useCourseTitle )
    ? Adapt.course.get('title')
    : Adapt.course.get('_navigationTitle').title;
   return {
     ...config,
     hideForMobile,
     title
   };
 } 
```

Reply:
Moved title logic to JS

Reply:
@swashbuck I haven't moved the `hideForMobile` / `isVisible` logic to the view / model as I'm mirroring the approach in core:

https://github.com/adaptlearning/adapt-contrib-core/blob/master/templates/header.jsx#L38